examples/ads.amp.html

```diff
@@ -401,6 +402,13 @@ <h2 id="atomx">Atomx</h2>
       data-id="1234">
   </amp-ad>
 
+  <h2 id="bidtellect">Bidtellect</h2>
```

Comment:
Hi, Sorry, I have been in bed with fever since Monday. I think I'm getting better now, Let me go trough this request.